.github/workflows/xpu-max1100.yml

```diff
@@ -34,47 +34,53 @@ jobs:
   unit-tests:
     runs-on: [self-hosted, intel, xpu]
     container:
-      image: intel/intel-extension-for-pytorch:2.1.20-xpu
+      image: intel/oneapi-basekit:2024.1.1-devel-ubuntu22.04
       ports:
         - 80
       options: --privileged -it --rm --device /dev/dri:/dev/dri -v /dev/dri/by-path:/dev/dri/by-path --ipc=host --cap-add=ALL
 
     steps:
     - uses: actions/checkout@v4
-    - name: Check container state
+    - name: Install prerequisite
       shell: bash
       run: |
-        ldd --version
-        python -c "import torch; print('torch:', torch.__version__, torch)"
-        python -c "import torch; import intel_extension_for_pytorch; print('XPU available:', torch.xpu.is_available())"
+        apt-get update
+        apt-get install clinfo libaio-dev python3-pip -y
+        pip install torch==2.1.0.post2 -f https://developer.intel.com/ipex-whl-stable-xpu
```

Comment:
Why do we need the versions from this index?

Reply:
Verisions from this index is closer to the release of intel_extension_for_pytorch and wheels can be downloaded stable and fast.

Reply:
Apologies for the slow response on this - would it be possible to get torch from the main pytorch index and leave the Intel specific wheels from developer.intel?